easybuild/easyblocks/g/gromacs.py

```diff
@@ -63,6 +63,8 @@ def extra_options():
         extra_vars.update({
             'double_precision': [None, "Build with double precision enabled (-DGMX_DOUBLE=ON), " +
                                  "default is to build double precision unless CUDA is enabled", CUSTOM],
+            'single_precision': [None, "Build with single precision enabled (-DGMX_DOUBLE=OFF), " +
```

Comment:
How come we don't just set this as default True? That's how they are used.

Reply:
Well single could  be set to True, but double precisions default depends on if CUDA is enabled or not.
If CUDA then double = False.

I was lazy and just copied the lines

Reply:
Hm, ok because
```
        precisions = []
        if self.cfg.get('single_precision') is None or self.cfg.get('single_precision'):
            precisions.append('single')
        if self.cfg.get('double_precision') is None or self.cfg.get('double_precision'):
            precisions.append('double')
```
didn't seem like there was any such check

Reply:
No, that check is somewhere else.